src/rgw/rgw_auth_s3.cc

```diff
@@ -223,8 +223,20 @@ namespace rgw {
 namespace auth {
 namespace s3 {
 
-/* FIXME(rzarzynski): duplicated from rgw_rest_s3.h. */
-#define RGW_AUTH_GRACE_MINS 15
+bool is_time_skew_ok(time_t t)
+{
+  auto req_tp = ceph::coarse_real_clock::from_time_t(t);
+  auto cur_tp = ceph::coarse_real_clock::now();
+
+  if (std::chrono::abs(cur_tp - req_tp) > RGW_AUTH_GRACE) {
```

Comment:
oops! the make check failure is a real one. `std::chrono::abs()` is a c++17ism, so we'll need to do this instead:
```c++
  if (req_tp < cur_tp - RGW_AUTH_GRACE ||
      req_tp > cur_tp + RGW_AUTH_GRACE) {
```

Reply:
Thanks, @cbodley! Done.